code/modules/recycling/disposal.dm

```diff
@@ -498,18 +498,27 @@
 		qdel(H)
 
 /obj/machinery/disposal/CanPass(atom/movable/mover, turf/target, height=0, air_group=0)
-	if (istype(mover,/obj/item) && mover.throwing)
+	if(ishuman(mover) && mover.throwing)
+		var/mob/living/carbon/human/H = mover
+		if(H.stats.getPerk(PERK_SPACE_ASSHOLE))
```

Comment:
Silly feature.

Reply:
Clever feature*(i made it, so its clever)